icicle-compiler/src/Icicle/Sea/FromAvalanche/Base.hs

```diff
@@ -91,7 +91,7 @@ mangleToSeaName = SeaName . Text.pack . concatMap mangle . show . pretty
      | c >= 'A' && c <= 'Z'
      = [c]
      | c == '_'
-     = "__"
+     = "_"
```

Comment:
Yeah actually that sounds easier.. unless there's something other than underscore we can use as an escape, which we know won't occur in the original string... `µµ`?